src/sidebar/components/ShareDialog/ImportAnnotations.tsx

```diff
@@ -59,6 +69,11 @@ function ImportAnnotations({
     [annotations, getDisplayName],
   );
 
+  // Try to pre-select current user
+  if (userList && !selectedUser) {
+    setSelectedUser(userList.find(user => user.userid === currentUser) ?? null);
+  }
```

Comment:
This has one problem. If selected user is not on the list, this block will run on every render, but it shouldn't have a very big impact, unless the list of users is huge.

Reply:
The most important thing is that the argument to the setter is strictly equal if nothing changed. Calling a state setter with a value that is strictly-equal to the current value is a no-op and cheap. If the value changes on each render, that would cause an infinite loop and be expensive.